src/classes/DateLiteralConstant_UnitTests.cls

```diff
@@ -0,0 +1,220 @@
+@isTest
+public class DateLiteralConstant_UnitTests {
```

Comment:
This class seems to have grown very large and is becoming difficult to maintain. Please consider refactoring to reduce the number of methods.